bgw-core/src/main/kotlin/tools/aqua/bgw/components/uicomponents/TextInputUIComponent.kt

```diff
@@ -32,6 +32,7 @@ import tools.aqua.bgw.visual.Visual
  * @param width Width for this [TextInputUIComponent].
  * @param height Height for this [TextInputUIComponent].
  * @param text Text for this [TextInputUIComponent].
+ * @param prompt Prompt for this [TextArea].
```

Comment:
Docs should reference TextInputUIComponent instead of TextArea. Might be a C&P error